src/env_configuration.sh

```diff
@@ -20,3 +20,6 @@ fi
 if [[ -z "$SIMPLE_OUTPUT" ]]; then
   SIMPLE_OUTPUT=$_DEFAULT_SIMPLE_OUTPUT
 fi
+if [[ -z "$STOP_ON_FAILURE" ]]; then
+  STOP_ON_FAILURE=$_DEFAULT_STOP_ON_FAILURE
```

Comment:
@Chemaclass when you merge this PR add this variable to the `.env.testing`

Reply:
I added it to `.env.example`